<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>MathEvoGen: GenesInSpace</title>
  <link rel="stylesheet" href="style.css" />
</head>
<body>

<div id="nav"></div>

<main>
  <h1>GenesInSpace</h1>
  <p> Rapid advances in sequencing have made an unprecedented amount of genomic data available from diverse organisms. 
    However, extracting useful information from this data, e.g., using sequences to <q>predict</q> individual phenotypes or make inferences about evolutionary history, remains challenging.  
    While several tools of genomic prediction, notably <b>Genome-Wide Association Studies (GWAS)</b>, have been developed in recent years, a common and unsolved challenge is to disentangle the 
    signatures of different evolutionary processes in genomic variation. In particular, genetic differences between groups of individuals may reflect either <b>natural selection</b>
    (typically on highly polygenic traits) or <b>spatial population structure</b> (i.e., limited exchange of individuals and genes between geographically separated groups) or both. 
    Moreover, both selection and population structure generate <b>correlations along the genome</b>, making it difficult to fine-map the individual genetic variants that affect traits.
  <br/>    <br/>  
The ERC-funded project <q>Genes in Space</q> aims to  develop new theoretical models to understand how natural selection acting on multiple genes in spatially extended populations shapes patterns of genetic variation 
    &mdash; both along the genome and across geographic space. Building upon this, we will explore to what extent we can  <i>disentangle</i>  selection and population structure in GWAS, and how this influences the <b>power</b>   
    of GWAS as well as its <b>portability</b>, i.e., the extent to which GWAS findings generalise to population groups with different ancestry. 
    The larger goal is to develop a more systematic understanding of the power and <i>limits</i> of sequence data for identifying functionally important regions of the genome, such as those that 
    influence disease risk in humans, yield in crops and environmental (mal-)adaptation in wild populations.
     </p>
 <img src="Schematic_ERC.png" alt="schematic" style="width:100%; height:auto; display: block;"> 
  
</main>

</body>

<script>
  fetch('nav.html')
    .then(response => response.text())
    .then(data => {
      document.getElementById('nav').innerHTML = data;
    });
</script>

</html>
