<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>MathEvoGen: Research</title>
  <link rel="stylesheet" href="style.css" />
</head>
<body>

<div id="nav"></div>

<main>
  <h1>Research</h1>
  <p> <b>Barriers to Gene Flow and Speciation</b>
  We are interested in the processes and genes that act as `barriers to gene flow' between populations as they evolve into distinct species.
  Specifically, we use models to understand how selection acting on multiple genes influences genetic and phenotypic differences between speciating populations, and to what extent genomic data can provide information about the 
   mechansims and genes involved in speciation.
  A key question here is: (when) do individual genes matter? Conversely, when does divergence between populations depend only on a few effective or parameters that capture the aggregate effect of many different barriers?
  This kind of approach  can contribute to an understanding of the limits of genomic inference by allowing us to ask:  what parameters and genetic details we can(not) infer from different kinds of data?
  </p>
  <p> <b>Linkage between genes and the response to selection</b>
 Quantitative genetics, which is the classical framework for describing short-term evolution, assumes that most traits are influenced by very many small-effect genetic variants that are unlinked, i.e., inherited independently of each other. 
    However, in most organisms, genes are physically linked into a handful of chromosomes, with large chunks of chromosome passed on intact from parent to offspring.
We are interested in understanding the consequences of  such linkage for the response to selection. 
    <!Specifically, how does linkage between large numbers of genes influence the rate of adaptation and the signatures it leaves in the genomes of populations?>
    A key question here is: does  selection `see' individual genetic variants or only large blocks of genome that  contain many such (beneficial and deleterious) variants. 
    Furthermore, can we identify selected blocks in genomic data, and what does this tell us about the strength or targets of selection?
  </p>
  <p> <b>Eco-evolutionary feedback and population extinction</b> </p>
Most models of evolutionary change assumes the size of evolving populations to be constant or determined by demographic processes independently of the genetics of the population. 
  However, this assumption breaks down in small or fragmented populations, which tend to accumulate harmful mutations at a higher rate,  which further reduces population fitness and growth,
  further increasing stochasticity and the accumulation of harmful variants. This can result in a positive feedback betweeen declining population sizes and increasing genetic load that may eventually lead to population extinction.
  We aim to better understand such eco-evolutionary feedback, using more realistic models that account for both genetic and demographic fluctations as well as realistic (polygenic) architectures of fitness variation. The larger goal is to   
quantitatively describe the conditions for the extinction of fragmented populations across a wide range of scenarios including both inbreeding and environmental maladaptation. 
</main>

</body>

<script>
  fetch('nav.html')
    .then(response => response.text())
    .then(data => {
      document.getElementById('nav').innerHTML = data;
    });
</script>

</html>
