<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>MathEvoGen: Research</title>
  <link rel="stylesheet" href="style.css" />
</head>
<body>

<div id="nav"></div>

<main>
  <h1>Research</h1>
  We are broadly interested in how patterns of genetic variation &mdash; across space and in time &mdash; are shaped by population structure, demography and polygenic selection. Specific research interests include:  
  <p> <b>Barriers to Gene Flow and Speciation</b>  </p>
  We are interested in the processes and genes that contribute to the breakdown of hybridisation and gene flow during speciation.
  Specifically, we use models to understand how selection acting on multiple genes influences genetic and phenotypic differences between speciating populations, and the extent to which patterns in genomic data can provide
  information about the mechanisms and genes involved in speciation.  
  A key question here is: (when) do individual genes matter? Conversely, when does divergence between populations depend only on a few <q>effective</q> or composite parameters that capture the combined effect of many different barriers to gene flow? 
  By identifying the parameters and genetic details that can(not) be inferred from data, we hope to gain a better understanding of the limits of genomic inference. More on this
  <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1010297" target="_blank" rel="noopener noreferrer">here </a>, 
  <a href="https://academic.oup.com/genetics/article/228/3/iyae140/7738845" target="_blank" rel="noopener noreferrer">here</a>,
  <a href="https://academic.oup.com/genetics/article/217/2/iyaa025/6042694" target="_blank" rel="noopener noreferrer">here </a>, and
  <a href="https://academic.oup.com/evolut/advance-article/doi/10.1093/evolut/qpaf047/8138001" target="_blank" rel="noopener noreferrer">here. </a>.
        <img src="EffectiveMig_schematic.png" alt="effectivemig" style="width:100%; height:auto; display: block;"> 
  <p> <b>Linkage between genes and the response to selection</b>  </p>
 Quantitative genetics, which is the classical framework for describing short-term evolution, assumes that traits are influenced by many small-effect genetic variants that are unlinked, i.e., inherited independently of each other. 
    However, in most organisms, genes are physically linked across a small number of chromosomes, with large chunks of chromosome passed on intact from parent to offspring.
We are interested in understanding the consequences of  such linkage for the response to selection. 
    <!Specifically, how does linkage between large numbers of genes influence the rate of adaptation and the signatures it leaves in the genomes of populations?>
    A key question here is: does  selection 'see' individual genetic variants or only large blocks of genome containing many such (beneficial and deleterious) variants. 
    Furthermore, can we identify selected blocks in genomic data, and what can we learn from them about the strength and targets of selection? More on this 
  <a href="https://academic.oup.com/genetics/article/209/4/1279/5931009" target="_blank" rel="noopener noreferrer">here</a> and <a href="https://academic.oup.com/genetics/article/210/4/1411/5931063" target="_blank" rel="noopener noreferrer"> here</a>.
        <img src="Linkage_schematic.png" alt="Linkage" style="width:100%; height:auto; display: block;"> 
  <p> <b>Eco-evolutionary feedback and population extinction</b> </p>
Most models of evolutionary change assume that the size of evolving populations is constant or determined by demographic processes, independently of the genetics of the population. 
  However, this assumption breaks down in small or fragmented populations, which tend to accumulate harmful mutations at a higher rate,  which reduces population fitness and size,
  which further accelerates the accumulation of harmful variants. This can result in a positive feedback betweeen declining population sizes and increasing genetic load that may eventually lead to population extinction.
  We aim to better understand such eco-evolutionary feedback using more realistic models that account for both genetic and demographic fluctations as well as realistic (polygenic) architectures of fitness variation. The larger goal is to   
quantitatively characterise the conditions for the extinction of fragmented populations across a wide range of scenarios, including both inbreeding and environmental maladaptation. 
  More on this <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/evo.14210" target="_blank" rel="noopener noreferrer"> here</a>,
  <a href="https://royalsocietypublishing.org/doi/full/10.1098/rstb.2021.0010" target="_blank" rel="noopener noreferrer"> here</a>, and 
  <a href="https://www.journals.uchicago.edu/doi/abs/10.1086/735562?journalCode=an" target="_blank" rel="noopener noreferrer"> here </a>.
    <img src="EcoEvo_schematic.png" alt="EcoEvo" style="width:100%; height:auto; display: block;"> 
In addition, we are starting various projects that focus on the genomic signatures of polygenic selection in spatially continuous populations,
  with the aim of better understanding if/how different evolutionary processes can be distinguished in genomic data. More information <a href="genes-in-space.html">here</a>. 
</main>

</body>

<script src="nav-highlight.js"></script>

</html>
